AmplifyTools/AmplifyXcode/Sources/AmplifyXcodeCore/Commands/AmplifyCommandError.swift

```diff
@@ -12,7 +12,7 @@ public struct AmplifyCommandError: Error {
         case unknown
         case fileNotFound
         case folderNotFound
-        case xcodeProject
+        case xcodeProject /// @see `XcodeProjectError`
```

Comment:
nit: style: we avoid trailing comments. Also, does a trailing doc-comment show up in the IDE associated with the correct member?

Reply:
No, it doesn't properly show up. Removed :)